### PR TITLE
Adds collect-resources.sh script

### PR DESCRIPTION
**Pull request by bolt12**
This script generates heap consumption graph for each deployed server and combines them all into one single file.

### CODE REVIEW

**Review by coot — APPROVED** [review]

File: scripts/process-resources.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          <$> readFile file
+  let processedContents = bimap (map (posixSecondsToUTCTime . fromIntegral)) 
+                                (map toGB) 
+                                contents
+      finalResult = uncurry (zipWith (\a b -> show a ++ "," ++ show b)) 
+                            processedContents
+
+  putStrLn (unlines finalResult)
+
+toGB :: Int -> Int
```
Comment (njd42) [inline scripts/process-resources.sh:25]: Technically - the SI unit of GiB is what you mean (GB are power of 10, GiB are powers of 2) see https://physics.nist.gov/cuu/Units/binary.html

**Review by njd42 — COMMENTED** [review]
Just a minor metrological nit - LGTM

**Comment by bolt12** [comment]
Thank you Neil!

**Comment by coot** [comment]
Can this be merged?

**Comment by bolt12** [comment]
Yes, sorry I forgot

**Pull request merged**